### PR TITLE
libXrender, new recipe

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: x11-libs/libxrender/libxrender-0.9.10.recipe [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+	devel:libX11$secondaryArchSuffix
+	devel:renderproto$secondaryArchSuffix
+	devel:xproto$secondaryArchSuffix
+	devel:xtrans$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	devel:util_macros$secondaryArchSuffix
+	cmd:aclocal
+	cmd:autoconf
+	cmd:libtool
```
Comment (fbrosson) [inline x11-libs/libxrender/libxrender-0.9.10.recipe]:
```diff
-	cmd:libtool
 	cmd:libtoolize$secondaryArchSuffix
```

File: x11-libs/libxrender/libxrender-0.9.10.recipe [hunk 2]
```diff
@@ -0,0 +1,78 @@
+SUMMARY="Library for the Render Extension to the X11 protocol"
+DESCRIPTION="The Xrender library is designed as a lightweight library \
+interface to the Render extension."
+HOMEPAGE="http://xorg.freedesktop.org/releases/individual/lib/"
```
Comment (fbrosson) [inline x11-libs/libxrender/libxrender-0.9.10.recipe]:
```diff
-HOMEPAGE="http://xorg.freedesktop.org/releases/individual/lib/"
+HOMEPAGE="https://xorg.freedesktop.org/releases/individual/lib/"
```

File: x11-libs/libxrender/libxrender-0.9.10.recipe [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+COPYRIGHT="2000 SuSE, Inc
+	2001,2003 Keith Packard"
+LICENSE="MIT (no promotion)"
+REVISION="1"
+SOURCE_URI="https://www.x.org/releases/individual/lib/libXrender-$portVersion.tar.gz"
+CHECKSUM_SHA256="770527cce42500790433df84ec3521e8bf095dfe5079454a92236494ab296adf"
+SOURCE_FILENAME="libXrender-$portVersion.tar.gz"
```
Comment (fbrosson) [inline x11-libs/libxrender/libxrender-0.9.10.recipe]:
```diff
-SOURCE_FILENAME="libXrender-$portVersion.tar.gz"
```
(not needed, as it's the default value)

BTW, the tar.bz2 is available.
And I've checked that the tar inside the bz2 is identical to the tar inside the gz, so you might wish to siwtch to it:
```diff
-SOURCE_URI="https://www.x.org/releases/individual/lib/libXrender-$portVersion.tar.gz"
+SOURCE_URI="https://www.x.org/releases/individual/lib/libXrender-$portVersion.tar.bz2"
-CHECKSUM_SHA256="770527cce42500790433df84ec3521e8bf095dfe5079454a92236494ab296adf"
+CHECKSUM_SHA256="c06d5979f86e64cabbde57c223938db0b939dff49fdb5a793a1d3d0396650949"
```

Comment (Begasus) [reply x11-libs/libxrender/libxrender-0.9.10.recipe]: Should we use bz2 over tar.gz? Always used the last ones, but if it's better I can easily switch :)

File: x11-libs/libxrender/libxrender-0.9.10.recipe (hunk 1 above)
Comment (Begasus) [reply x11-libs/libxrender/libxrender-0.9.10.recipe]: bugger .. :)

File: x11-libs/libxrender/libxrender-0.9.10.recipe (hunk 3 above)
Comment (fbrosson) [reply x11-libs/libxrender/libxrender-0.9.10.recipe]: It's not _better_, it's just _smaller_ :)

(BTW, the reason why some people prefer to stick to gz is that some OSes don't have bzip2.)

File: x11-libs/libxrender/libxrender-0.9.10.recipe [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+	"
+
+defineDebugInfoPackage libxrender$secondaryArchSuffix \
+	$libDir/libXrender.so.$libVersion
+
+BUILD()
+{
+	autoreconf -vfi
+	runConfigure ./configure
+}
```
Comment (alaviss) [inline x11-libs/libxrender/libxrender-0.9.10.recipe]: make?

**Pull request merged**